packages/plugin-print/src/index.js

```diff
@@ -1,8 +1,12 @@
 import Path from "path";
+import Url from 'url';
 import bMFont from "load-bmfont";
 import { isNodePattern, throwError } from "@jimp/utils";
 import { measureText, measureTextHeight, splitLines } from "./measure-text";
 
+const __filename = url.fileURLToPath(import.meta.url);
```

Comment:
Does all compiled output have the correct code here?

ex:

- cjs
- esm
- browser

Once verified will merge